vercel.json

```diff
@@ -3221,6 +3226,11 @@
       "destination": "/docs/concepts/why-clickhouse-is-so-fast",
       "permanent": true
     },
+    {
```

Comment:
Not sure where this is coming from but there is no such page. BFloat16 has always been on the same page as Floats.